gems/pending/VMwareWebService/MiqVimVm.rb

```diff
@@ -878,11 +879,30 @@ def getScsiCandU
       all_unit_numbers = [*0..MAX_SCSI_DEVICES]
```

Comment:
Not related to your change, but good to change along the way:

Shall we move this out of the loop? Ruby probably already optimizing this, but still move out of the loop would make it clear that this is just a constant set.